src/test/scala/com/cognite/sdk/scala/v1/TransformationsTest.scala

```diff
@@ -0,0 +1,214 @@
+        conflictMode = Some("upsert"),
+        Some(false),
+        sourceOidcCredentials = Some(credentials.toFlatCredentials),
+        destinationOidcCredentials = Some(credentials.toFlatCredentials),
+        externalId = shortRandomUUID(),
+        ignoreNullFields = true,
+        dataSetId = Some(216250735038513L)
+      )
+    }
+    val resCreates = client.transformations.createItems(Items(transformationsToCreate))
+    val externalIds = transformationsToCreate.map(_.externalId)
+
+    try {
+      val resFilterName = client.transformations
+        .filter(TransformationsFilter(nameRegex = Some("transformation-sdk")))
+        .compile
+        .toList
+      resFilterName.size shouldBe transformationsToCreate.size
+      resCreates.map(_.name) should contain theSameElementsAs resFilterName.map(_.name)
+
+      val resFilterQueryRegex = client.transformations
+        .filter(TransformationsFilter(queryRegex = Some("select abc")))
+        .compile
+        .toList
+      resFilterQueryRegex.size >= transformationsToCreate.size shouldBe true
+      resFilterQueryRegex.map(_.query).forall(q => q.startsWith("select abc")) shouldBe true
+
+      val resFilterUpsertEvents = client.transformations
+        .filter(
+          TransformationsFilter(destinationType = Some("events"), conflictMode = Some("upsert"))
+        )
+        .compile
+        .toList
+      resFilterUpsertEvents.size >= transformationsToCreate.size shouldBe true
+      resFilterUpsertEvents
+        .map(_.destination.asInstanceOf[GeneralDataSource].`type`)
+        .toSet shouldBe Set("events")
+      resFilterUpsertEvents.map(_.conflictMode).toSet shouldBe Set("upsert")
+
+      val max1DaysAgo = TimeFilter(max = Some(Instant.now().minus(1, ChronoUnit.DAYS).toEpochMilli))
+      val resFilterCreatedTime1DayAgo = client.transformations
+        .filter(
+          TransformationsFilter(
+            nameRegex = Some("transformation-sdk"),
+            createdTime = Some(max1DaysAgo)
+          )
+        )
+        .compile
+        .toList
+      resFilterCreatedTime1DayAgo.isEmpty shouldBe true
+
+      val min3MinsAgo =
+        TimeFilter(min = Some(Instant.now().minus(3, ChronoUnit.MINUTES).toEpochMilli))
+      val resFilterCreatedTime3MinsAgo = client.transformations
+        .filter(
+          TransformationsFilter(
+            nameRegex = Some("transformation-sdk"),
+            createdTime = Some(min3MinsAgo)
+          )
+        )
+        .compile
+        .toList
+      resFilterCreatedTime3MinsAgo.size shouldBe transformationsToCreate.size
+
+      val resFilterDataSetId = client.transformations
+        .filter(
+          TransformationsFilter(
+            nameRegex = Some("transformation-sdk"),
+            dataSetIds = Some(Seq(CogniteInternalId(216250735038513L)))
+          )
+        )
+        .compile
+        .toList
+      resFilterDataSetId.size shouldBe transformationsToCreate.size
+    } finally
```

Comment:
We always use braces for those (both `finally`, `catch`, and `if` etc.), see https://github.com/cognitedata/cognite-sdk-scala/blob/master/src/test/scala/com/cognite/sdk/scala/v1/EventsTest.scala#L133

Reply:
Still missing for `finally`, I'm afraid. ;)